legal/association_statute.md

```diff
@@ -0,0 +1,224 @@
+
+§10 Stowarzyszenie może organizować i brać udział w akcjach oraz kampaniach społecznych, seminariach, pokazach, imprezach edukacyjnych i kulturalnych, organizować konkursy oraz przyznawać nagrody i stypendia, przeprowadzać akcje medialne (kampanie informacyjne i akcje społeczne) promujące działania lub cele Stowarzyszenia.
+
+§11 Stowarzyszenie może upowszechniać w internecie, w środkach masowego przekazu, w formie audycji radiowych lub telewizyjnych, filmów telewizyjnych, artykułów prasowych oraz reportaży, wywiadów w prasie, radio i telewizji, i za pomocą wydawnictw wiedzy w zakresie objętym celami Stowarzyszenia.
+
+§12 Stowarzyszenie realizuje swoje cele poprzez prowadzenie odpłatnej i nieodpłatnej działalności pożytku publicznego w zakresie wskazanym w §4 oraz art. 4 ust. 1 pkt. 11-13 w Ustawie o działalności pożytku publicznego i wolontariacie z dnia 24 kwietnia 2003 wraz z późniejszymi zmianami.
+
+Rozdział IV: Wolontariat i Współpraca
+
+§13 Wolontariuszami Stowarzyszenia mogą być osoby o różnych kompetencjach, w tym: programiści, testerzy oprogramowania, specjaliści w zakresie użyteczności aplikacji, menedżerowie projektów, aktywiści społeczni, pracownicy organizacji non-profit, badacze społeczni, prawnicy i inni, których wiedza oraz umiejętności mogą przyczynić się do realizacji celów Stowarzyszenia.
+
+§14 Stowarzyszenie może współpracować z innymi podmiotami prowadzącymi działalność wspomagającą lub obejmującą cele Stowarzyszenia.
+
+§15 Dla wspierania realizacji swoich celów statutowych Stowarzyszenie może przystępować do związków, stowarzyszeń oraz innych organizacji społecznych, zawiązywać fundacje, których cele będą zbieżne z celami Stowarzyszenia, samodzielnie lub przy udziale innych osób fizycznych, osób prawnych oraz jednostek organizacyjnych nieposiadających osobowości prawnej.
+
+Rozdział V: Członkowie
+
+§16 Stowarzyszenie posiada członków:
+a) zwyczajnych,
+b) wspierających.
+
+§17 Członkiem Stowarzyszenia mogą być osoby fizyczne i prawne. Osoba prawna może być jedynie członkiem wspierającym.
+
+§18
+1. Członkiem zwyczajnym Stowarzyszenia może być każda osoba fizyczna, która złoży pisemną deklarację i zostanie zarekomendowana przez dwóch członków Stowarzyszenia.
+2. Przyjęcia nowych członków zwyczajnych dokonuje Zarząd uchwałą podjętą nie później niż w ciągu miesiąca od daty złożenia deklaracji.
+
+§19
+1. Członkiem wspierającym może zostać osoba fizyczna i prawna, która złoży pisemną deklarację pomocy finansowej, rzeczowej i/lub merytorycznej w realizacji celów Stowarzyszenia.
+2. Przyjęcia nowych członków wspierających dokonuje Zarząd uchwałą podjętą nie później niż w ciągu 3 miesięcy od daty złożenia deklaracji.
+
+§20 Członkowie zwyczajni mają prawo do:
+a) biernego i czynnego uczestnictwa w wyborach do władz Stowarzyszenia,
+b) udziału we wszystkich aktywnościach realizowanych przez Stowarzyszenie,
+c) zgłaszania wniosków dotyczących działalności Stowarzyszenia.
+
+§21 Członkowie zwyczajni mają obowiązek:
```

Comment:
Nie jestem pewien, tutaj nie powinno być:
`§21 Członkowie wspierający mają obowiązek:`
?

Reply:
W 20 masz prawa zwyczajnych, w 21 masz obowiązki. w 22 jest o prawach wspierających. w 23 o ich obowiązkach.

powinno mieć to sens.

Reply:
Ta wątpliwość chyba rozwiązana. Można było uporządkować albo wg klucza prawa/obowiązki albo członek zwyczajny/wspierający. Tu poszliśmy pierwszym torem, ale merytorycznie chyba wszystko się zgadza